crates/matrix-sdk-crypto/Cargo.toml

```diff
@@ -50,7 +50,7 @@ zeroize = { version = "1.3.0", features = ["zeroize_derive"] }
 
 [target.'cfg(target_arch = "wasm32")'.dependencies.ruma]
 version = "0.6.1"
-features = ["client-api-c", "js", "rand", "unstable-msc2676", "unstable-msc2677"]
+features = ["client-api-c", "js", "rand", "unstable-msc2676", "unstable-msc2677", "signatures"]
```

Comment:
```suggestion
features = ["client-api-c", "js", "rand", "signatures", "unstable-msc2676", "unstable-msc2677"]
```

Reply:
Fixed in https://github.com/matrix-org/matrix-rust-sdk/pull/716/commits/1223191f17eeb7559b1701b1445033a91053af39.